tools/bin/hawq

```diff
@@ -120,6 +120,23 @@ def main():
         if second_arg not in cluster_type_list:
             print STOP_HELP
             sys.exit(1)
+        # Prints deprecation warning when using old syntax "hawq stop <object> -u" to reload GUC
+        if len(sub_args.split("-")) > 1:
+            if "-u" in sub_args or "--reload" in sub_args or "u" in sub_args.split("-")[1]:
+                deprecationMessage = """
+                hawq stop <object> -u is being deprecated and replaced by 'hawq reload <object>'
```

Comment:
This deprecation message is not align with other messages.
Run 'hawq stop cluster -u' then you can see.